components/cards/cards.css

```diff
@@ -0,0 +1,112 @@
+.main-card{
+    max-width: 300px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    background-color: white;
+    position: relative;
+    border-radius: 0.5rem;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+    box-sizing: border-box;
+    margin: 1.2rem;
+}
+
+.card-badge{
+    position: absolute;
+    top: 1.2rem;
+    background-color:#1FABD7;
+    color: white;
+    padding: 0rem 1rem;
+}
+
+.card-dismiss{
+    position: absolute;
+    top: 5px;
+    right: 10px;
+    color: white;
+    font-size: 1.2rem;
+}
+
+.card-dismiss:hover{
+    cursor: pointer;
+}
+
+.img-comp{
```

Comment:
- you could have given a better name - something like card-img or img-header 
- also if you set top-border radius to image, it is breaking in one of the cards(the one which does not have description - 5th one)
   - you can set "overflow: hidden" in the parent container